config/deploy.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+# config valid for current version and patch releases of Capistrano
+lock '~> 3.19.1'
```

Comment:
I believe we've usually removed the `lock` from our `config/deploy.rb` files. I can only find one project where we didn't.